### DISCUSSION TITLE
Question about eddy detection

### DISCUSSION THREAD

**Discussion by Louis-Bonhomme** [AntSimi/py-eddy-tracker#85] [Q&A] [answered]
Good afternoon,

I have a question about the condition over the number of pixels in the contour to be identified as an eddy:
the number of pixels is for the surface enclosed by the contour, or is it the number of pixels on the circumference of the contour?

Thank you in advance.

**Comment by AntSimi** [comment, marked as answer]
Hi,
To be exact algorithm count center of pixel in contour.
[Code here](https://github.com/AntSimi/py-eddy-tracker/blob/e934346b0d0a6b5e0e6d0cea722926c153a9f21c/src/py_eddy_tracker/dataset/grid.py#L1203)

**Reply by Louis-Bonhomme** [reply]
Thank you
